docs/vector/README.md

```diff
@@ -0,0 +1,161 @@
+# Using Vector with k8ssandra-operator
+
+Since k8ssandra-operator v1.5.0, it is possible to scrape Cassandra metrics with the Vector agent and customize its configuration to add transformers and sinks.
+
+More information about Vector can be found in the [official documentation](https://vector.dev/docs/).
+
+## Enabling Vector agent
+
+To enable Vector agent, you need to add a `.spec.cassandra.telemetry` section in the `K8ssandraCluster` manifest:
```

Comment:
Note: as for other pieces in the `telemetry` section, you can configure Vector cluster-wide and override things at DC level. Both levels will be merged in the `Coalesce` function.

Reply:
I'll add a sentence on this 👍